apps/prairielearn/elements/pl-drawing/elements.py

```diff
@@ -87,10 +97,12 @@ def generate(el, data):
             "offset_y": offset_y,
         }
 
+    @staticmethod
     def is_gradable():
```

Comment:
```suggestion
    def is_gradable() -> bool:
```

Reply:
Resolving so we can discuss on https://github.com/PrairieLearn/PrairieLearn/pull/11855#discussion_r2064280008.